infrastructure/template.yaml

```diff
@@ -676,83 +595,92 @@ Outputs:
 
   PrivateApiGatewayId:
-    Description: "API GatewayID of the private Address CRI API"
-    Value:
-      !If [
-        IsNotDevEnvironment,
-        !Ref PrivateNinoCheckApi,
-        !Ref DevOnlyNinoCheckApi,
-      ]
+    Description: API GatewayID of the private Address CRI API
+    Value: !If
+      - IsNotDevEnvironment
+      - !Ref PrivateNinoCheckApi
+      - !Ref DevOnlyNinoCheckApi
     Export:
       Name: !Sub ${AWS::StackName}-PrivateApiGatewayId
 
   CurrentTimeFunction:
-    Description: "Fetch Current Time Function Arn"
+    Description: Fetch Current Time Function ARN
     Value: !GetAtt CurrentTimeFunction.Arn
 
   CurrentTimeFunctionRoleArn:
-    Description: "Fetch Current Time Function Role Arn"
+    Description: Fetch Current Time Function Role ARN
     Value: !GetAtt CurrentTimeFunctionRole.Arn
 
   CreateAuthCodeFunction:
-    Description: "Create Authorization Code Function ARN"
+    Description: Create Authorization Code Function ARN
     Value: !GetAtt CreateAuthCodeFunction.Arn
 
   CreateAuthCodeFunctionRoleArn:
-    Description: "Create Authorization Code Function Arn"
+    Description: Create Authorization Code Function ARN
     Value: !GetAtt CreateAuthCodeFunctionRole.Arn
 
+  # Delete when this is not used
   TimeFunction:
-    Description: "Time Lambda Function ARN"
+    Description: Time Lambda Function ARN
+    Value: !GetAtt TimeFunction.Arn
+
+  TimeFunctionArn:
+    Description: Time Lambda Function ARN
     Value: !GetAtt TimeFunction.Arn
 
   TimeFunctionRoleArn:
-    Description: "TimeFunctionRole Arn"
+    Description: TimeFunctionRole ARN
     Value: !GetAtt TimeFunctionRole.Arn
 
+  # Delete when this is not used
   MatchingFunction:
-    Description: "Matching Lambda Function ARN"
+    Description: Matching Lambda Function ARN
+    Value: !GetAtt MatchingFunction.Arn
+
+  MatchingFunctionArn:
+    Description: Matching Lambda Function ARN
     Value: !GetAtt MatchingFunction.Arn
 
   MatchingFunctionRoleArn:
-    Description: "MatchingFunctionRole Arn"
+    Description: MatchingFunctionRole ARN
     Value: !GetAtt MatchingFunctionRole.Arn
 
   NinoCheckStateMachineArn:
-    Description: "NinoCheck StateMachine Arn"
-    Value: !GetAtt NinoCheckStateMachine.Arn
+    Description: NINO Check state machine ARN
+    Value: !Ref NinoCheckStateMachine
 
   CheckSessionStateMachineArn:
-    Description: "Session Check StateMachine Arn"
-    Value: !GetAtt CheckSessionStateMachine.Arn
+    Description: Session Check state machine ARN
+    Value: !Ref CheckSessionStateMachine
 
   NinoIssueCredentialStateMachineArn:
-    Description: "Nino Issue Credential StateMachine Arn"
-    Value: !GetAtt NinoIssueCredentialStateMachine.Arn
+    Description: NINO Issue Credential state machine ARN
+    Value: !Ref NinoIssueCredentialStateMachine
 
   CommonStackName:
-    Description: "Common Stack Name"
+    Description: Common Stack Name
     Value: !Ref CommonStackName
 
+  # Delete when this is not used
```

Comment:
These "delete" comments, you can easily test this by removing and running the integration tests. If they pass, you can remove them.